### PR TITLE
chore(backend): Spelling

### PR DESCRIPTION
**Pull request by jsoref**
### SUMMARY

Subset of #19648 per https://github.com/apache/superset/pull/19648#pullrequestreview-939431828

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: docs/docs/miscellaneous/chart-params.mdx
Comment (jsoref) [inline docs/docs/miscellaneous/chart-params.mdx:21]: I'm happy to update the table, but I'd rather wait until I get a general acceptance of the changes.

File: superset/reports/schemas.py
Comment (jsoref) [inline superset/reports/schemas.py:87]: As noted in https://github.com/apache/superset/pull/19648#discussion_r847405605, I'm not generally doing grammar fixes, this was just in the neighborhood for the line.

**Comment by rusackas** [comment]
Sorry this has fallen through the cracks for so long @jsoref - would you mind rebasing this? These are some pretty embarrassing typos, so I hope we can (finally) get this reviewed and merge it.

**Comment by jsoref** [comment]
@rusackas: done. Note that I'm not rechecking, it's just a naive rebase. If you merge the current PR(s), I can rerun to catch things that were introduced in the interim.

File: helm/superset/Chart.yaml
Comment (rusackas) [inline helm/superset/Chart.yaml:32]: Not sure if this change is intentional...

File: superset/connectors/sqla/models.py
Comment (rusackas) [inline superset/connectors/sqla/models.py:758]:
```suggestion
        # todo(yongjie): create a physical table column type in separate PR
```

File: superset/examples/configs/datasets/examples/FCC_2018_Survey.yaml
Comment (rusackas) [inline superset/examples/configs/datasets/examples/FCC_2018_Survey.yaml:933]: @betodealmeida would the changes in this file pose any potential danger to anyone's workflows? Legit spelling fixes, but want to make sure this yaml change doesn't cause fallout.

**Review by rusackas — APPROVED** [review]
LGTM. Approving to unblock, since I looked through it all and it _seems_ fine, but would just like a second opinion on one risk factor from @betodealmeida or someone they refer me to!

File: docs/docs/miscellaneous/chart-params.mdx
Comment (rusackas) [reply docs/docs/miscellaneous/chart-params.mdx:21]: I guess we're there now :D

**Comment by rusackas** [comment]
Looks like this just needs a little more love...
* Just one conflict
* A table update?
* looks like `helpers.py` didn't have an instance of `allows_hidden_ordeby_agg` get updated to `allows_hidden_orderby_agg` which is causing some errors on CI.

File: docs/docs/miscellaneous/chart-params.mdx
Comment (jsoref) [inline docs/docs/miscellaneous/chart-params.mdx:75]: This table was already broken wrt alignment, so I'm not going to fix it -- if someone wants to lint all the markdown in this repository, I'll leave that to them for a distinct change (this is already quite large)

File: superset/connectors/sqla/models.py
Comment (jsoref) [inline superset/connectors/sqla/models.py:758]: I've added an `a` as well

**Comment by rusackas** [comment]
This PR needs a little rebase, but also needs code owner approval from @craig-rueda @dpgaspar or @villebro

File: superset/examples/configs/datasets/examples/FCC_2018_Survey.yaml
Comment (villebro) [inline superset/examples/configs/datasets/examples/FCC_2018_Survey.yaml:1323]: I think we need to double check that this doesn't break examples creation, as I assume this is the actual name of the column in the dataset..

**Review by villebro — COMMENTED** [review]
Woah, I had no idea we had this many spelling errors laying around. Thanks for the fixes, but wondering if some of those CI errors aren't legit, especially the dataset column names.

File: superset/examples/configs/datasets/examples/FCC_2018_Survey.yaml
Comment (dpgaspar) [reply superset/examples/configs/datasets/examples/FCC_2018_Survey.yaml:1323]: good catch, yes it is the column name

File: docs/static/resources/openapi.json
Comment (dpgaspar) [inline docs/static/resources/openapi.json:564]: this needs to be changed here: https://github.com/apache/superset/blob/master/superset/charts/schemas.py#L109 and then regenerate the OAS, the spec should not be edited directly

File: superset/examples/configs/datasets/examples/FCC_2018_Survey.yaml
Comment (dpgaspar) [reply superset/examples/configs/datasets/examples/FCC_2018_Survey.yaml:933]: Just checked it's the column name on the examples, will prob cause some impact

File: docs/static/resources/openapi.json
Comment (jsoref) [reply docs/static/resources/openapi.json]: For reference, that *was* changed in this PR by the same commit.

However, in the interest of not dealing w/ this mess at this time, I'm dropping all commits that touch this file. We can deal w/ it in a distinct PR, or perhaps someone else can.

**Comment by rusackas** [comment]
@dpgaspar the files that caused some concern in this PR have been reverted to their original state. I think everything is working as expected now and CI is passing, if you (or any other code owner reading this) can give it a final check.

**Comment by rusackas** [comment]
@jsoref we got the frontend one merged, thanks for all your hard work! I think this one is ready to go if it can be rebased to mitigate the last conflicts. There also might be some issue with the python linting, but I'm having a hard time fixing it since my local python env is _not happy_ with the pre-commit hook at the moment. Maybe the rebase will magically fix things 🤞

**Comment by jsoref** [comment]
@rusackas: thanks. I ran into someone's git hooks earlier today -- I *hate* hooks. So you definitely have my sympathy and appreciation.

File: superset/databases/decorators.py
Comment (eschutho) [inline superset/databases/decorators.py:58]: changing this value can affect people's metrics if they actively watching this value. That said, we don't have strict semver guidelines around logging and breaking changes, and I don't think that this value is used much. I'm probably ok with letting this slip through, but curious if anyone else has concerns. We can also wait until 3.0, but you'll likely have merge conflicts with these files if you wait that long to merge. Or you can just remove this change.

File: superset/examples/configs/charts/Messages_per_Channel.yaml
Comment (eschutho) [inline superset/examples/configs/charts/Messages_per_Channel.yaml:53]: This is actually a value in the data. If we change this, it will break the example chart.

File: superset/examples/configs/dashboards/Slack_Dashboard.yaml
Comment (eschutho) [inline superset/examples/configs/dashboards/Slack_Dashboard.yaml:254]:
```suggestion
    embedd-dashboards: "#ACE1C4"
```

File: superset/examples/multiformat_time_series.py
Comment (eschutho) [inline superset/examples/multiformat_time_series.py:101]: I think this value has been deprecated, but if you want to be safe, you can change it to `col.database_expression`

**Review by eschutho — CHANGES_REQUESTED** [review]
This is a great help. I added a few small notes.

**Comment by rusackas** [comment]
Re-requesting @eschutho's review... I believe the requested changes have been addressed (or reversed, in some cases)

**Review by eschutho — APPROVED** [review]

**Comment by rusackas** [comment]
Thanks @eschutho! I think we still need @dpgaspar / @villebro / @craig-rueda to approve this as code owners.

**Review by villebro — APPROVED** [review]

**Pull request merged**
